src/dataflow/src/decode/avro.rs

```diff
@@ -66,7 +66,7 @@ impl DecoderState for AvroDecoderState {
             }
             Err(err) => {
                 self.events_error += 1;
-                Err(format!("avro deserialization error: {}", err))
+                Err(format!("avro deserialization error: {:#}", err))
```

Comment:
Does this default to `{:#?}` Debug printing? Is there a difference between `{:#}`, `{:?}`, and `{:#?}`?

Reply:
Yes, there's a difference - `{:#}` enables the "alternate style"; `{:?}` enables debug printing, and `{:#?}` enables "alternate style debug printing".

What this actually means is up to the individual debug/display impls; for `anyhow::Error`, it is described here: https://docs.rs/anyhow/1.0.40/anyhow/struct.Error.html#display-representations